base_exception/tests/test_base_exception.py

```diff
@@ -2,43 +2,116 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 from odoo.tests import common
+from odoo import api, fields, models
+from .common import init_test_models
 
 import logging
 
 _logger = logging.getLogger(__name__)
 
 
-# @common.at_install(False)
```

Comment:
so this trick plays nice only if you run the whole test suite at install I guess?

Reply:
This is the way I've seen it being done in https://github.com/OCA/mis-builder/blob/10.0/mis_builder/tests/test_kpi_data.py#L19, and seems to be no issue testing during install, which is the default method and easiest to verify.

Reply:
hmm then I guess that if for any reason we'll need to run a test after installation we could use a different class and make sure this one is loaded before